engines/python/src/main/java/ai/djl/python/engine/Connection.java

```diff
@@ -120,10 +126,71 @@ CompletableFuture<Output> send(Input input) throws InterruptedException {
         return f;
     }
 
-    static String[] getPythonStartCmd(PyEnv pyEnv, Model model, int workerId, int port) {
+    static String[] getPythonStartCmd(
+            PyEnv pyEnv, Model model, int workerId, int port, int clusterSize) {
         Device device = model.getNDManager().getDevice();
         int deviceId = device.getDeviceId();
         int tensorParallelDegree = pyEnv.getTensorParallelDegree();
+        // int pipelineParallelDegree = pyEnv.getPipelineParallelDegree();
+        logger.info("Printing mpi boolean: {}", pyEnv.isMpiMode());
+
+        if (clusterSize > 1) {
+            String leaderAddress = Utils.getenv("LWS_LEADER_ADDRESS");
```

Comment:
need revisit all env names

Reply:
These names are fixed by LWS env, we cannot change this in LWS+EKS env.